core/src/modules/configLoaders/index.ts

```diff
@@ -129,8 +128,24 @@ export async function processConfigObjects(
       configObjects = sortConfigurationObjects(configObjects);
     }
   } catch (error) {
-    // the config loaders below will produce better error messages
-    sortError = error;
+    errors.push(error);
+    // If the sorting issue was that we couldn't find an object, the assumption
+    // is that the developer is targeting an object of some ID that does not
+    // exist.
+    if (error.message.includes("Cannot find ")) {
```

Comment:
maybe `startsWith` since are assuming that in the next line.

Reply:
because we control this code, we could also set something on the error like `unknownNodeId` or whatever and not have funny parsing.

Reply:
We could also collect all the unknown nodes in the topological sort so we tell them all the missing nodes